src/client/app/helpers/splash.screen.ts

```diff
@@ -0,0 +1,30 @@
+import { environment } from './environment';
+import { Strings } from '../strings';
+let { config } = PLAYGROUND;
+
+let strings = Strings();
+
+export function instantiateRibbon() {
```

Comment:
I wonder if the id of the element (or the dom element itself) should be passed in (just for future flexibility's sake; what if I need to call the element something else?)